scripts/variantstore/tieout/compare_data.py

```diff
@@ -261,9 +271,9 @@ def compare_sample_data(e1, e2):
                 log_difference(key, e1, e2) 
 
         # TODO: temporary until we decide what to do with spanning deletions
```

Comment:
is this #TODO comment still applicable?

Reply:
yep -- tied to https://github.com/broadinstitute/dsp-spec-ops/issues/143, but I'll add that to the comment as well